### DISCUSSION TITLE
Is Stackstorm Death? no Release since a Year?

### DISCUSSION THREAD

**Discussion by daBONDi** [StackStorm/st2#6093] [Q&A] [answered]
Is Stackstorm Death? No Release since a Year?

**Comment by nzlosh** [comment, marked as answer]
No, not dead.  The last commit was two weeks ago https://github.com/StackStorm/st2/commit/80cd7bc50ab90a9d09f1a42f03e530010bc331bd and minutes from last TSC meeting https://github.com/StackStorm/community/issues/129

**Comment by arm4b** [comment]
That's just in time, we were preparing `v3.8.1` which is shipped today: https://github.com/StackStorm/st2/releases/tag/v3.8.1
@daBONDi  Join us in Slack, it'll be easier to get an idea about the development and if you want to help, - there are some interesting tasks to contribute to.

**Comment by 0megam** [comment]
We lookin to utilize StackStorm in our infrastructure, but looks like new version has not been release for a long time.
Is project still alive ?

**Reply by nzlosh** [reply]
Yes, it's still alive. https://github.com/StackStorm/st2/commits/master/
